operators/samples/instance.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: tenant-john-doe
+  labels:
+    crownlabs.polito.it/operator-selector: local
```

Comment:
At the moment, this is not required, as the check is performed on the namespace and not the instance itself